caldp/file_ops.py

```diff
@@ -9,6 +9,44 @@
 from caldp import log
 
 
+def get_input_path(input_uri, ipppssoot, make=False):
+    """Fetches the path to input files"""
+    cwd = os.getcwd()
+    if input_uri.startswith("file"):
+        input_path = input_uri.split(":")[-1]
+    else:
+        input_path = os.path.join(cwd, "inputs", ipppssoot)
+        if make is True:
+            os.makedirs(input_path, exist_ok=True)
+    return input_path
+
+
+def append_trailer(input_path, output_path, ipppssoot):
+    """Fetch process log and append to trailer file
+    Note: copies trailer file from inputs directory
+    and copies to outputs directory prior to appending log
+    """
+    try:
+        tra1 = list(glob.glob(f"{output_path}/{ipppssoot.lower()}.tra"))
+        tra2 = list(glob.glob(f"{output_path}/{ipppssoot.lower()[0:5]}*.tra"))
+        if os.path.exists(tra1[0]):
+            trailer = tra1[0]
+        elif os.path.exists(tra2[0]):
+            trailer = tra2[0]
+        else:
+            log.error("Trailer file not found - skipping.")
```

Comment:
can you make the missing trailer an info instead of error?

Reply:
yes

Reply:
And I think if it's truly missing, what we want to do is create it, though we would want to tag up with Violet on that